content/pages/schedule.rst

```diff
@@ -25,7 +25,7 @@ Sessions
 
 .. raw:: html
 
-     <iframe width='800' height='450' frameborder='0' src='https://docs.google.com/spreadsheets/d/1jMR61C1wbPEMAg95cL5rjKrUOQgVkI0iuCpjo_xYLSY/pubhtml?gid=203045195&amp;single=true&amp;widget=true&amp;headers=false'></iframe>
+    <iframe width='800' height='450' frameborder='0' src="https://docs.google.com/spreadsheets/d/e/2PACX-1vTzrSCQ-hNGUIDZMlCK5W7V2czDOSGKyApKmVxzqdsGLYQjLVfmn5dGo8OyJAfoosuq6H354M4HWsth/pubhtml?gid=1229700711&amp;single=true&amp;widget=true&amp;headers=false"></iframe>
```

Comment:
Why changing this to a different Google doc? I'd prefer to keep this in the same doc just with a new tab.

Reply:
@ramereth It was the same document, but I changed the publish setting so that it includes the tabs from previous years.